docs/features/account-portal.mdx

```diff
@@ -0,0 +1,60 @@
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+
+
+# Introduction
+
+The Account Portal offers a comprehensive solution for managing user authentication and profile management in your web application and is the fastest way to add authentication to your application with minimal code required. 
+
+<Image />
+
+
+# Getting Started
+
+To use the Account Portal, simply add Clerk to your application and protect your pages. Any time an unauthenticated user attempts to access a protected page in your application, we will automatically redirect them to the Account Portal pages to Sign in.
```

Comment:
nit, remove adverbs that trivialize effort, and prefer to reference Clerk in the third-person, instead of "we":

```suggestion
To use the Account Portal, add Clerk to your application and protect your pages. Any time an unauthenticated user attempts to access a protected page in your application, they will automatically get redirected to the Account Portal pages to Sign in.
```